README.md

```diff
@@ -96,12 +96,13 @@ import 'server-only'
 
 import type {QueryParams} from '@sanity/client'
 import {createClient, groq} from 'next-sanity'
+import {draftMode} from 'next/headers'
 
 const projectId = process.env.NEXT_PUBLIC_SANITY_PROJECT_ID // "pv8y60vp"
 const dataset = process.env.NEXT_PUBLIC_SANITY_DATASET // "production"
 const apiVersion = process.env.NEXT_PUBLIC_SANITY_API_VERSION // "2023-05-03"
 
-const client = createClient({
+const sanityClient = createClient({
```

Comment:
```suggestion
const client = createClient({
```

This should continue to be `client` as that's what we use in our starter templates and most of our educational material 😄